packaging/mingw-w64/configure-build-msys2-mingw.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+$mingw64 cmake -DCMAKE_MAKE_PROGRAM=mingw32-make -G "MinGW Makefiles" \
+      -DCMAKE_INSTALL_PREFIX=$PWD/dest/ \
+      -DMINGW_BUILD:BOOL=ON \
+      -DWITHOUT_WIN32_CONFIG:BOOL=ON \
+      -DRDKAFKA_BUILD_EXAMPLES:BOOL=ON \
+      -DRDKAFKA_BUILD_TESTS:BOOL=ON \
+      -DWITH_LIBDL:BOOL=OFF \
+      -DWITH_PLUGINS:BOOL=OFF \
+      -DWITH_SASL:BOOL=ON \
+      -DWITH_SSL:BOOL=ON \
+      -DWITH_ZLIB:BOOL=OFF \
+      -DRDKAFKA_BUILD_STATIC:BOOL=OFF \
+      -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS:BOOL=TRUE .
+
+$mingw64 mingw32-make
+$mingw64 mingw32-make install
+
+export PATH="$PWD/dest/bin:/mingw64/bin/:${PATH}"
```

Comment:
out of curiosity, why is this PATH update needed for the test-runner?

Reply:
`$PWD/dest/bin` is for librdkafka.dll and librdkafka++.dll. `/mingw64/bin/` is for the MinGW runtime outside of the mingw64 shell. (on the other test runs you're setting `LD_LIBRARY_PATH` for this stuff)

Reply:
Ah, ok, so PATH is used for dynamic library lookups on mingw (or is it windows in general)? Didnt know that.

Reply:
Correct, Windows in general uses `PATH` to find dlls. Unlike *nix it also always searches the current working directory.